src/main/java/budgetbuddy/MainApp.java

```diff
@@ -112,6 +116,28 @@ private LoansManager initLoansManager(Storage storage) {
         }
     }
 
+    /**
+     * Loads and returns an Accounts Manager from storage.
+     * Returns an empty Accounts Manager if no file found or if exception occurs during loading.
+     */
+    private AccountsManager initAccountsManager(Storage storage) {
+        Optional<AccountsManager> accountsManagerOptional;
+
+        try {
+            accountsManagerOptional = storage.readAccounts();
+            if (accountsManagerOptional.isEmpty()) {
+                logger.info("Accounts file not found. Will be starting with an empty AccountsManager.");
+            }
+            return accountsManagerOptional.orElseGet(AccountsManager::new);
+        } catch (DataConversionException e) {
+            logger.warning("Accounts file not in the correct format. Will be starting with an empty AccountsManager.");
+            return new AccountsManager();
+        } catch (IOException e) {
+            logger.warning("Problem while reading from accounts file. Will be starting with an empty AccountsManager.");
+            return new AccountsManager();
+        }
+    }
```

Comment:
Does this mean the default account will be cleared as well? We should still initialise with a default account right?